torch/distributed/distributed_c10d.py

```diff
@@ -73,7 +76,20 @@ def __new__(cls, name):
                              "Gloo or MPI backend for collective operations "
                              "on CPU tensors.")
         elif value == Backend.UNDEFINED:
-            raise ValueError("Invalid backend: '{}'".format(name))
+            try:
```

Comment:
Can we add some docs to this method to explain the the new feature?

Reply:
I add it in latest commit, pls help check it out